CONTRIBUTING.md

```diff
@@ -41,9 +41,12 @@ $ npm run start
 
 ## Pull Requests
 
-We actively welcome your pull requests.
+We actively welcome your pull requests. 
 
-1. Fork the repo and create your branch from `master`.
+Netlify CMS uses the [Forking Workflow](https://www.atlassian.com/git/tutorials/comparing-workflows#forking-workflow) + [Feature Branches](https://www.atlassian.com/git/tutorials/comparing-workflows#feature-branch-workflow). Additionally, PR's should [rebase on rather than merge](https://www.atlassian.com/git/tutorials/merging-vs-rebasing) master to stay current. 
+
+1. Fork the repo
+2. create a feature branch from `master`. If you're working on an issue, prefix your branch name with {{issue_number}}_description, otherwise use xx_description.
```

Comment:
I'm not certain we want to go here, not a ton of value in branch naming conventions to make it worth requiring of the community. Maybe when community involvement grows. I *do* intend to implement conventional changelog, which will require formatted commit messages, so I'd like to cut down on the number of particulars we're enforcing.

Reply:
I think its good to make a recommendation. Enforcement can be lax. As a reviewer who reviews code on half a dozen projects, it's immensely helpful to have something more than a text description in a  branch name.  

Documents at the end of a day are an ideal. What happens in practice it a matter of enforcement for the maintainers.